### ISSUE TITLE
VSIX Validation Error Dependent Task Inputs

### ISSUE THREAD

**Issue by charleszipp** [charleszipp/azure-pipelines-tasks-terraform#35]
The following error is being reported when trying to upload new vsix created for #19 
```
Task definition input 'ensureBackend' and Task definition input 'backendAzureRmResourceGroupLocation' must belong to same group as they are dependent inputs for Task with ID '721c3f90-d938-11e8-9d92-09d7594721b5'.
```

**Comment by charleszipp** [comment]
released with 0.4.1

**Comment by tplive** [comment]
Hi! We would like to help build this feature, just want to know if someone is already working on it? If so, we can collaborate, if not, we can get started on requirements.. 😄
